src/pcapi/core/bookings/api.py

```diff
@@ -140,6 +140,13 @@ def mark_as_unused(booking: Booking) -> None:
     repository.save(booking)
 
 
+def unvalidate_bookings(bookings: typing.List[Booking]) -> typing.List[Booking]:
```

Comment:
"unvalidate" n'a pas l'air d'être dans le dictionnaire. :)

```suggestion
def invalidate_bookings(bookings: typing.List[Booking]) -> typing.List[Booking]:
```

... ceci dit, j'ai peur qu'on introduise un nouveau terme ("invalidate") pour une action qui existe déjà ("mark_as_unused"). Est-ce qu'on veut vraiment ajouter cette fonction ? L'autre possibilité serait de reporter ce cette boucle `for` dans le code de l'appelant (`edit_stock()`).

Reply:
Ok, pour le "unvalidate" (encore une tentative inconsciente de mettre du français partout)
On s'est effectivement posé la question d'où mettre la boucle, on va regarder du côté du edit_stock pour voir